notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -78,6 +86,41 @@ internal class NotifiqueListView(
     setHasFixedSize(true)
     adapter = listAdapter
     addItemDecoration(DividerItemDecoration(context.getDrawable(R.drawable.divider)!!))
+
+    selectionTracker = SelectionTracker.Builder(
+        "list-selection-id",
+        this,
+        object : ItemKeyProvider<Long>(SCOPE_MAPPED) {
+          override fun getKey(position: Int): Long? {
+            return listAdapter.getKey(position)
+          }
+
+          override fun getPosition(key: Long): Int {
+            val index = listAdapter.currentList!!.indexOfFirst {
+              key == it.id
+            }
+            return if (index == -1) NO_POSITION else index
+          }
+        },
+        object : ItemDetailsLookup<Long>() {
+          override fun getItemDetails(e: MotionEvent): ItemDetails<Long>? {
+            val childView = findChildViewUnder(e.x, e.y) ?: return null
+            val viewHolder = getChildViewHolder(childView)
+            val position = viewHolder.adapterPosition
+            return object : ItemDetails<Long>() {
+              override fun getSelectionKey(): Long? {
+                return listAdapter.getKey(position)
+              }
+
+              override fun getPosition(): Int {
+                return position
+              }
+            }
+          }
+        },
+        StorageStrategy.createLongStorage()
```

Comment:
[This documentation](https://developer.android.com/reference/androidx/recyclerview/selection/SelectionTracker.Builder.html) says that the the long storage does not support "band selection." I'm not sure what that means, and I don't see why it is different than the string implementation from the source code. I thought perhaps the documentation was out of date, but I didn't see significant changes in the git history.

Reply:
I'm going to assume band selection is dragging selection, and it does work.